packages/isomorphic/README.md

```diff
@@ -0,0 +1,100 @@
+# @xrplf/isomorphic
+
+A collection of isomorphic implementations of crypto and utility functions.
+
+Browser implementations of cryptographic functions use `@noble/hashes` and `crypto` for node .
```

Comment:
Why not just use `@noble/hashes` for everything? Is `crypto` a native node package?

Reply:
Yes `crypto` is native node package.